docs/networking.md

```diff
@@ -99,11 +98,35 @@ Here's an example Compose file defining several networks. The `proxy` service is
         options:
           foo: "1"
           bar: "2"
+
+## Using externally-created networks
+
+You can use the `external` option to indicate that a network has been created outside of Compose. In the example below, `proxy` is the gateway to the outside world, via a network called `outside` which is expected to already exist.
+
+    version: 2
+
+    services:
+      proxy:
+        build: ./proxy
+        networks:
+          - outside
+          - default
+      app:
+        build: ./app
+        networks:
+          - default
+
+    networks
       outside:
-        # The 'outside' network is expected to already exist - Compose will not
-        # attempt to create it
         external: true
 
+You can also specify the name of the network separately from the name used to refer to it within the Compose file:
+
+    networks
+      outside:
+        external:
+          name: actual-name-of-network
```

Comment:
What do you think about moving this section to the reference to document `external` , and have a section here that talks about changing the defualt network using `external` instead?